src/Microsoft.AspNetCore.Mvc.ViewFeatures/Controller.cs

```diff
@@ -195,7 +195,6 @@ public virtual PartialViewResult PartialView(object model)
         [NonAction]
         public virtual PartialViewResult PartialView(string viewName, object model)
         {
-            // Do not override ViewData.Model unless passed a non-null value.
```

Comment:
Ah, lookie that. I accidently did the PartialVeiw comment and not the View one. :+1: